hardware/index.md

```diff
@@ -242,9 +241,9 @@ main computer.
 | Core variant: | [ARM Cortex-M0+](https://www.arm.com/products/processors/cortex-m/cortex-m0plus.php)
 | Flash ROM     | 128KB
 | RAM           | 16KB
-| Speed         | 16MHz
+| Speed         | [16Mhz (crystal)](https://github.com/ARMmbed/DAPLink/blob/master/source/hic_hal/freescale/kl26z/MKL26Z4/system_MKL26Z4.c#L69) 48MHz (max)
```

Comment:
If this is linked to the master branch it can change overtime and the line could have different content, will be better to use a tag in the URL.

Reply:
I've linked to the latest v0254 branch